test/BasePaymentTest.php

```diff
@@ -206,7 +206,7 @@ public function createBasket(): Basket
      *
      * @return Card
      */
-    protected function createCardObject(string $cardNumber = '5453010000059543'): Card
+    protected function createCardObject(string $cardNumber = '4711100000000000'): Card
```

Comment:
Missing in the changelog
e.g. "Fixed failing card tests."

Reply:
Added entry.